src/test/java/com/epam/tc/hw7/listeners/TestNGListener.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.tc.hw7.listeners;
+
+/**
+ * Created by Roman Iovlev on 14.02.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import static com.epam.jdi.light.driver.ScreenshotMaker.takeScreen;
+import static com.epam.jdi.light.settings.WebSettings.TEST_NAME;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+import static com.epam.jdi.tools.LinqUtils.last;
+import static java.lang.System.currentTimeMillis;
+
+import com.epam.jdi.tools.Safe;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import org.testng.IInvokedMethod;
+import org.testng.IInvokedMethodListener;
+import org.testng.ITestNGMethod;
+import org.testng.ITestResult;
+import org.testng.annotations.Test;
+
+public class TestNGListener implements IInvokedMethodListener {    private Safe<Long> start = new Safe<>(0L);
```

Comment:
Please, fix formatting before push the file

Reply:
not fixed

Reply:
I took this listener from the JDI framework and it comes with this format